import-scripts/dmp-import-vars-functions.sh

```diff
@@ -319,35 +319,37 @@ function waitOutDmpGermlineServerInstabilityPeriod() {
 
 # Function for consuming fetched samples after successful import
 function consumeSamplesAfterSolidHemeImport {
+    drop_dead_instant_string=$(date --date="+2hours +50minutes" -Iseconds) # nearly 3 hours from now
     if [ -f $MSK_IMPACT_CONSUME_TRIGGER ] ; then
         waitOutDmpTumorServerInstabilityPeriod
         echo "Consuming mskimpact tumor samples from cvr"
-        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -c $MSK_IMPACT_PRIVATE_DATA_HOME/cvr_data.json
+        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -c $MSK_IMPACT_PRIVATE_DATA_HOME/cvr_data.json -z $drop_dead_instant_string
         waitOutDmpGermlineServerInstabilityPeriod
         echo "Consuming mskimpact germline samples from cvr"
-        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -g -c $MSK_IMPACT_PRIVATE_DATA_HOME/cvr_gml_data.json
+        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -g -c $MSK_IMPACT_PRIVATE_DATA_HOME/cvr_gml_data.json -z $drop_dead_instant_string
         rm -f $MSK_IMPACT_CONSUME_TRIGGER
     fi
     if [ -f $MSK_HEMEPACT_CONSUME_TRIGGER ] ; then
         waitOutDmpTumorServerInstabilityPeriod
         echo "Consuming mskimpact_heme samples from cvr"
-        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -c $MSK_HEMEPACT_PRIVATE_DATA_HOME/cvr_data.json
+        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -c $MSK_HEMEPACT_PRIVATE_DATA_HOME/cvr_data.json -z $drop_dead_instant_string
         rm -f $MSK_HEMEPACT_CONSUME_TRIGGER
     fi
     if [ -f $MSK_ACCESS_CONSUME_TRIGGER ] ; then
         waitOutDmpTumorServerInstabilityPeriod
         echo "Consuming mskaccess samples from cvr"
-        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -c $MSK_ACCESS_PRIVATE_DATA_HOME/cvr_data.json
+        $JAVA_BINARY $JAVA_CVR_FETCHER_ARGS -c $MSK_ACCESS_PRIVATE_DATA_HOME/cvr_data.json -z $drop_dead_instant_string
         rm -f $MSK_ACCESS_CONSUME_TRIGGER
     fi
 }
 
 # Function for consuming fetched samples after successful archer import
 function consumeSamplesAfterArcherImport {
+    drop_dead_instant_string=$(date --date="+1hour" -Iseconds) # one hour from now
```

Comment:
Ben asked below about some timeouts that are 2 hours instead of nearly 3.  This one is 1 hour.  Maybe you explain somewhere.

Reply:
explanation and adjustments added above in Benjamin's comment. Timeouts have been largely standardized to 3 hours per fetcher execution now, with an end-to-end timeout for the entire fetch script set to 12 hours.